responsibleai/tests/test_model_analysis.py

```diff
@@ -61,7 +61,7 @@ def test_model_analysis_iris(self, manager_type):
         }
 
         for model in models:
-            run_model_analysis(model, X_train, X_test, LABELS, [],
+            run_model_analysis(model, X_train, X_test, LABELS, None,
```

Comment:
Should we test both ways, though (at least once, if not on all of these tests)?

Reply:
Is there a use case when user might pass an empty list for categorical_features?